dashboard/test/controllers/api/v1/pd/applications_controller_test.rb

```diff
@@ -597,6 +598,27 @@ class ApplicationsControllerTest < ::ActionController::TestCase
       )
     end
 
+    test 'cohort view returns facilitator applications of correct statuses' do
+      expected_applications = []
+      facilitator_cohort_view_statuses = Api::V1::Pd::ApplicationsController::COHORT_VIEW_STATUSES & FACILITATOR_APPLICATION_CLASS.statuses
+      FACILITATOR_APPLICATION_CLASS.statuses.each do |status|
+        application = create FACILITATOR_APPLICATION_FACTORY, course: 'csp'
+        application.update_column(:status, status)
```

Comment:
Is there a reason we're using `update_column` here instead of passing status to the factory?
```rb
create FACILITATOR_APPLICATION_FACTORY, course: 'csp', status: status
```

Reply:
We have `after_initialize` and `before_create` [hooks in the model](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/pd/application/application_base.rb#L106) that set the initial status to 'unreviewed', so passing the status in to the factory doesn't actually determine the initial status.